src/main/java/org/gridsuite/dynamicmargincalculation/server/RestResponseEntityExceptionHandler.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.dynamicmargincalculation.server;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@ControllerAdvice
+public class RestResponseEntityExceptionHandler {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RestResponseEntityExceptionHandler.class);
+
+    @ExceptionHandler(DynamicMarginCalculationException.class)
+    protected ResponseEntity<Object> handleDynamicSecurityAnalysisException(DynamicMarginCalculationException exception) {
```

Comment:
```suggestion
    protected ResponseEntity<Object> handleDynamicMarginCalculationException(DynamicMarginCalculationException exception) {
```

Reply:
DONE in https://github.com/gridsuite/dynamic-margin-calculation-server/pull/1/commits/5c376744edd9fa5afe1e568d675b18068e80f938